### PR TITLE
Add node to node clone functional test

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Adds a functional test to verify we can clone from one node to another node when using local volume provisioner storage.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/cloner_test.go [hunk 1]
```diff
@@ -84,6 +84,88 @@ var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:compo
...
+				}
+				pvNode := pv.Spec.NodeAffinity.Required.NodeSelectorTerms[0].MatchExpressions[0].Values[0]
+				if ok, val := nodeMap[pvNode]; ok && val {
+					nodeMap[pvNode] = false
+					By("Labeling PV " + pv.Name + " as source")
+					sourcePV = &pv
+					if sourcePV.GetLabels() == nil {
+						sourcePV.SetLabels(make(map[string]string))
+					}
+					sourcePV.GetLabels()["source-pv"] = "yes"
```
Comment (mhenriks) [inline tests/cloner_test.go]: are labels guaranteed to be non nil?

Comment (awels) [reply tests/cloner_test.go]: I made sure right above that they are not nil
```
if sourcePV.GetLabels() == nil {
	sourcePV.SetLabels(make(map[string]string))
}
```

Comment (mhenriks) [reply tests/cloner_test.go]: LOL

Comment (mhenriks) [reply tests/cloner_test.go]: what's up with the crio failure?

Comment (awels) [reply tests/cloner_test.go]: No idea, haven't had a chance to look at what is failing yet.

Comment (awels) [reply tests/cloner_test.go]: The crio lane has a non local volume PV without a NodeAffinity which is causing the npe. Fixed.

**Pull request merged**